packages/Pagination/styles.jsx

```diff
@@ -0,0 +1,144 @@
+import styled from 'styled-components'
+import { media } from '@tds/core-responsive'
```

Comment:
Please add `@tds/core-responsive` to `dependencies` in this component's package.json.